config/typescript.js

```diff
@@ -1,20 +1,79 @@
 module.exports = {
   parser: '@typescript-eslint/parser',
+  parserOptions: {
+    'sourceType': 'module',
+  },
   plugins: [
     '@typescript-eslint',
   ],
-  extends: [
-    'eslint:recommended',
-    'plugin:@typescript-eslint/eslint-recommended',
-    'plugin:@typescript-eslint/recommended',
-  ],
   rules: {
+    // Checked by TypeScript - ts(2378)
```

Comment:
Is there somewhere these redundant rules are listed?

Reply:
I copied these from the plugin config itself, I don't believe there is a canonical list